xhr.bs

```diff
@@ -1021,6 +1025,12 @@ method must run these steps:
   <p>Otherwise, if the <a>synchronous flag</a> is set, run these substeps:
 
   <ol>
+   <li>
+    <p>If <a>context object</a>'s <a>relevant settings object</a> has a
+    <a>responsible document</a> and it is <em>not</em> <a>allowed to use</a> the
+    <code><a>sync-xhr</a></code> feature, then run <a>handle response
```

Comment:
If this is meant to be a string I think it should be `"<code><a>sync-xhr</a></code>"` here and below. That's how you write down strings per Infra.

Also, no wrapping inside phrasing elements.

Reply:
Oh, also, since the `<p>` is the only child you need to do `<li><p>...`.

Reply:
All fixed